jib-core/src/main/java/com/google/cloud/tools/jib/event/events/LogEvent.java

```diff
@@ -89,4 +91,24 @@ public Level getLevel() {
   public String getMessage() {
     return message;
   }
+
+  @VisibleForTesting
+  @Override
+  public boolean equals(Object obj) {
```

Comment:
tiny nit: s/obj/other/ to avoid using abbreviations. The `other` below can be `otherLogEvent`.